[gh-pages][ZEPPELIN-2055] Add "What's new in 0.7" and Twitter feed to website main page

AhyoungRyu opened the pull request:
### What is this PR for?
There are lots of improvement in upcoming release 0.7(as mentioned in #1972 (release note)). To emphasize them to users and make contents enrich Zeppelin official website, I added some description \w gif images. And also added Twitter news feed at the bottom of the main page. 

### What type of PR is it?
gh-pages | Improvement

### Todos
* [x] - need to take care of gif images' alignment when screen width is small 

### What is the Jira issue?
[ZEPPELIN-2055](https://issues.apache.org/jira/browse/ZEPPELIN-2055)

### How should this be tested?
Build `gh-pages` branch as described in [here](https://github.com/apache/zeppelin/tree/gh-pages#build-website). Or see the below gif sreenshot img :) 

### Screenshots (if appropriate)
 - Before 
![before](https://cloud.githubusercontent.com/assets/10060731/22618943/ee52b5ea-eb2c-11e6-896a-f90a50f41998.gif)

 - After 
![after](https://cloud.githubusercontent.com/assets/10060731/22623919/66aeaa22-ebae-11e6-8835-d5680fe61697.gif)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

Leemoonsoo commented:
Looks great!

What do you think add things about improvements on Python/pyspark in what's new section?

Matplotlib integration in python/pyspark and conda support in python are something worth to mention IMO.

felixcheung commented:
^^ that should be a per release what's new page, I think?

felixcheung commented on _includes/themes/zeppelin/_navigation.html:
```diff
@@ -70,9 +70,13 @@
+    <div class="border row">
+      <div class="col-md-12 col-sm-12 col-xs-12 text-center newBtn">
+        <a href="http://youtu.be/_PQbVH_aO5E" target="_zeppelinVideo" class="btn btn-primary btn-lg bigFingerButton" role="button">Tutorial Video</a>
+        <a href="./download.html" class="btn btn-primary btn-lg bigFingerButton" role="button">GET ZEPPELIN 0.7</a>
```
this is nice - should it just say "GET LATEST ZEPPELIN NOW"? so we don't have to keep updating this ;)

felixcheung commented on index.md:
```diff
@@ -18,10 +18,15 @@ limitations under the License.
 {% include JB/setup %}
 <br />
+<h1 class="index-header">
+  What is Apache Zeppelin ?
```
do we need the extra ` ` behind Zeppelin

felixcheung commented:
looks good!

AhyoungRyu commented:
@Leemoonsoo @felixcheung Thanks for your review! As @Leemoonsoo mentioned I added "Improvement in Python" section like below. How about this?
<img width="500" alt="screen shot 2017-02-05 at 1 47 32 pm" src="https://cloud.githubusercontent.com/assets/10060731/22623929/c9e52c74-ebae-11e6-92b9-15642604d5fe.png">

@felixcheung Addressed your comments as well. And it'll be better to update "What's new ~ " section for every major release. Surely I'm willing to do that :)
<img width="500" alt="screen shot 2017-02-05 at 2 28 44 pm" src="https://cloud.githubusercontent.com/assets/10060731/22623949/7184cc28-ebaf-11e6-9ac9-94d35bece497.png">

felixcheung commented:
cool!

Leemoonsoo commented:
@AhyoungRyu Great!

nit:

While this will be seen from not only existing Zeppelin user but also people new to Zeppelin, how about change "Interpreter Per User / Note" to something like "Multi-user support improvement", more general words. And address "Interpreter per user /note" and "interpreter impersonation" in the text below?

And in "Personalized Note Mode", would it be possible to somehow mention 'Collaboration' mode, although it's not really a new feature? Something like "Switch between Collaboration and Personalized mode". So people new to Zeppelin can easily understand that there're two possible modes switchable "Collaboration" and "Personalized". What do you think?

AhyoungRyu commented:
@Leemoonsoo Thanks for the detailed suggestion. So I added some description about "Interpreter impersonation" under "Multi-user Support Improvement" and mentioned about "Collaboration mode"   . How about this? 
<img width="700" alt="screen shot 2017-02-05 at 9 25 04 pm" src="https://cloud.githubusercontent.com/assets/10060731/22626117/9adf11f2-ebe9-11e6-969e-c141a891f2aa.png">

Feel free to suggest if you have better wording or sentences in your mind :)

Leemoonsoo commented:
LGTM!

felixcheung commented:
LGTM

AhyoungRyu commented:
Will merge into `gh-pages` if there are no more comments!
